cmake/Findlibbacktrace.cmake

```diff
@@ -0,0 +1,19 @@
+include(GNUInstallDirs)
```

Comment:
we should prepare this dependency using the recipe machinery. `Findfoo.cmake` is supposed to *find* a library not for building it.